guides/players/radios.md

```diff
@@ -73,7 +73,7 @@ To change the block channel you'll need to detach the 343's Handle (as shown in
 <hr/>
 
 # AN/PRC-148 Usage
-The AN/PRC-148 is a long range intra-squad level radio carried and used most commonly by [Team Leaders](/guides/players/teamlead.md) to communicate with one another.
+The AN/PRC-148 is a long range intersquad radio carried and used most commonly by [Team Leaders](/guides/players/teamlead.md) to communicate with one another.

```

Comment:
`intra-squad` or `squad level`

Reply:
tf is wrong with u

Reply:
Inter squad would still be acceptable as its between two groups not within one group

Reply:
?

Reply:
The 148 is within a squad though, not between squads

Reply:
`intra-squad` or `inter-team` would work